packages/shell/src/Adapter/index.tsx

```diff
@@ -2,7 +2,6 @@ import {
   Connectors,
```

Comment:
I would not update the shell on this branch. Just increment the version of the core-sdk package when publishing and change this in another PR with the purpose of updating the core-sdk adapter and implement the network change request.